pandas/core/indexes/base.py

```diff
@@ -2971,9 +2971,12 @@ def _convert_listlike_indexer(self, keyarr, kind=None):
 
         Returns
         -------
-        tuple (indexer, keyarr)
-            indexer is an ndarray or None if cannot convert
-            keyarr are tuple-safe keys
+        indexer, keyarr
+            The return can be:
+            - indexer : an ndarray or None if cannot convert
+            - keyarr : tuple-safe keys
+
+            Return indexer, keyarr.
```

Comment:
Can you check in this diff the previous change? That's how a Return with multiple values is documented.